.travis.yml

```diff
@@ -1,29 +1,27 @@
+sudo: false
 language: python
 env:
   - TOXENV=flake8-py27
   - TOXENV=flake8-py33
   - TOXENV=py26-dj14
   - TOXENV=py27-dj14
-  - TOXENV=py26-dj15
```

Comment:
We don't **need to**... but for me it's a scope thing. We have limited time to support the project, and, despite the fact that **commercial** bodies use/rely on the project, nobody is going to jump in with an offer to support development — so we **do need** to draw the line tightly. IMO. Discuss. :-)